docs/release/release-process.md

```diff
@@ -140,6 +171,29 @@ The `master` branch can now be opened for new features since we have a safe
 release branch to continue bug fixes and improvements for the release itself.
 Essentially, `master` is free to now diverge from the release branch.
 
+### Release Branch Prep
+
+In the core Crossplane repository, we need to update the release branch docs and
+examples to point to the new versions that we will be releasing soon.
+
+* Documentation, such as [Installation
+  instructions](https://github.com/crossplane/crossplane/blob/release-0.9/docs/install.md#installing-infrastructure-providers),
+  and
+  [Stack](https://github.com/crossplane/crossplane/blob/release-0.9/docs/stack.md)
+  and
+  [App](https://github.com/crossplane/crossplane/blob/release-0.9/docs/app.md)
+  guides.
+  * searching for `:master` will help a lot here
+  * `kubectl -k` statements will contain `?ref=master`, which should be updated
+    to the release branch name instead, e.g. `?ref=release-0.4`
```

Comment:
These have been removed so one less thing to update :tada:

Reply:
Nice, I've removed this note about `?ref=master` hits in the docs during release branch prep